skafka/src/main/scala/com/evolutiongaming/skafka/Blocking.scala

```diff
@@ -1,14 +1,30 @@
 package com.evolutiongaming.skafka
 
+import cats.effect.{Async, ContextShift}
+import cats.implicits._
+import com.evolutiongaming.catshelper.FromFuture
+
 import scala.concurrent.{ExecutionContext, Future}
 
-trait Blocking[F[_]] {
-  def apply[T](f: => T): F[T]
-}
+final case class Blocking(ec: ExecutionContext)
```

Comment:
why do we need this case class at all ?

Reply:
The idea was to make the passing of blocking `ExecutionContext` more "type safe" so those who provide it (especially as an implicit) need to wrap it to "blocking" explicitly. Pretty much as we use value classes.

Reply:
https://github.com/ChristopherDavenport/linebacker ?

Reply:
Refactored `Blocking` as we discussed.